lib/src/command/outdated.dart

```diff
@@ -32,11 +32,12 @@ class OutdatedCommand extends PubCommand {
   String get docUrl => 'https://dart.dev/tools/pub/cmd/pub-outdated';
 
   OutdatedCommand() {
-    argParser.addOption('format',
-        help: 'Defines how the output should be formatted. Defaults to color '
-            'when connected to a terminal, and no-color otherwise.',
-        valueHelp: 'FORMAT',
-        allowed: ['color', 'no-color', 'json']);
+    argParser.addFlag('color',
+        help: 'Whether to color the output. Defaults to color '
+            'when connected to a terminal, and no-color otherwise.');
+
+    argParser.addFlag('json',
+        help: 'Outputs the results in a json formatted report');
```

Comment:
This needs `negatable: false` specifying `--no-json` makes no sense.

Reply:
Good point.
Done.